### PR TITLE
feat: updates to video review GUI

### PR DESCRIPTION
**Pull request by sven337**
My commit history is super ugly because I made 0 effort to clean it up due to the fast iteration going on at the moment.
Let me know what you'd rather see - what I cannot promise is super well written hyper detailed commit messages, but I can certainly squash and summarize?
I do not expect this to go on for long - this is nearly done wrt the basics I require for my own use case, later development will happen based on community feedback if any.

tl;dr Lots of small quality of life updates to the app.

### CODE REVIEW

**Review by Benehiko — APPROVED** [review]
Sorry for not getting back to this sooner. This looks quite! I think we can just squash merge the PR and it'll be fine.
